### PR TITLE
CI/CD: publish docker image in GHCR

### PR DESCRIPTION
**Pull request by ivard**

### CODE REVIEW

**Comment by bobhageman** [comment]
Wouldn't it be nice to also publish to Docker Hub? Free for public images. I see there is a [privacybydesign](https://hub.docker.com/u/privacybydesign) account, is that ours?

**Comment by ivard** [comment]
I've no idea. Are you sure it is free? When I initially looked it looked like only personal accounts are free. And I don't want to publish anything using a personal account.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - E-mail address revalidation, addressing issues where user's e-mail addresses can be (temporary) invalid
+- Publish the Docker image of the `irma`` CLI tool on ghcr.io/privacybydesign/irma
```
Comment (bobhageman) [inline CHANGELOG.md:11]: Superfluous backtick

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,12 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ### Changed
 - Use separate application user in Dockerfile for entrypoint
 - Rename RevocationStorage's UpdateLatest function to LatestUpdates. This name better fits its behaviour. The functionality stays the same.
 - Validate revocation witness before revocation update is applied
 - RevocationStorage's EnableRevocation function does not return an error anymore if it has been enabled already
+- Use a scratch Docker image as base for the Dockerfile
```
Comment (bobhageman) [inline CHANGELOG.md:18]: Would change it to something like:
Use a from scratch Docker image as base for the Dockerfile
or
Use a Docker image created from scratch as base for the Dockerfile

**Review by bobhageman — APPROVED** [review]

**Review by sjoerdlmkns — APPROVED** [review]

**Pull request merged**